src/pages/iou/steps/IOUAmountPage.js

```diff
@@ -216,8 +203,15 @@ class IOUAmountPage extends React.Component {
             .value();
     }
 
+    navigateToCurrencySelectionPage() {
+        Navigation.navigate(this.props.hasMultipleParticipants
+            ? ROUTES.getIouBillCurrencyRoute(this.props.reportID)
+            : ROUTES.getIouRequestCurrencyRoute(this.props.reportID));
```

Comment:
```suggestion
        Navigation.navigate(
        	this.props.hasMultipleParticipants
            ? ROUTES.getIouBillCurrencyRoute(this.props.reportID)
            : ROUTES.getIouRequestCurrencyRoute(this.props.reportID)
            );
```
Something like this.